package.json

```diff
@@ -224,4 +224,4 @@
       "npm run lint:fix"
     ]
   }
-}
+}
```

Comment:
can you revert this change

Reply:
I have reverted back but i don't know why it is showing space here. 227 is the last line in my json file with }